REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.3
-Nettle 0.2
-JSON 0.5
-ZMQ 0.3
-Compat 0.7.15
-Conda 0.1.5
+Compat 0.8.0
+Nettle 0.2.2
+JSON 0.5.2
+ZMQ 0.3.2
```

Comment:
I have come to realize that the compatibility of the dependent packages.
I was not mind carefully enough to that. I'll take care.
I found these lower bounds within the commit.
Compat.jl : v0.7.20 for is_apple, is_windows
ZMQ.jl :  v0.3.1 to fix FDWatcher error during test